views/view_package.dt

```diff
@@ -124,10 +150,21 @@ block body
 			- if( auto pl = "copyright" in versionInfo )
 				li#copyright= pl.get!string
 
+		- if (isMine)
+			//- TODO: make .actions not conditional on isMine, instead have actions which are always there here too (either as a: button.large for actions or p.detail for descriptions)
```

Comment:
Does this actually make sense to display a greyed "Manage this package" button?

Reply:
no not a grayed "Manage this package" button, the button should only be visible if logged in and as the author, but the `.actions` div there should be always visible once we have another action